pkg/xds/generator/admin_proxy_generator.go

```diff
@@ -84,10 +90,31 @@ func (g AdminProxyGenerator) Generate(ctx context.Context, _ *core_xds.ResourceS
 		return nil, err
 	}
 
+	readinessCluster, err := envoy_clusters.NewClusterBuilder(proxy.APIVersion, dppReadinessClusterName).
+		Configure(envoy_clusters.ProvidedEndpointCluster(
+			govalidator.IsIPv6(adminAddress),
+			core_xds.Endpoint{
+				UnixDomainPath: core_xds.DppReadinessSocketName(proxy.Metadata.WorkDir, proxy.Id.ToResourceKey().Name),
```

Comment:
Do you know if it works correctly on Windows? It's changing current behavior and I wonder if that won't break the flow (if anyone is doing healthcheck)?

Reply:
I think it does not. NamedPipe is the similar technical on Windows, but the DP is not listening named pipe. Do you think we need to support Windows? We are using UnixDomain listeners on MeshMetrics and many other features. When considering backporting, I agree we need to support Windows.

Reply:
When we do backporting, we can just respond a literal string "READY" on the `admin_proxy_generator.go` and don't need this component at all. Let me create a new PR to do that and let's discuss if that's more easy to be backported.
The reason why I created this PR introducing the component was because we can include more factors when calculating the readiness of `kuma-sidecar`. 
However, they will only be introduced in the future, I think. And in those version, we may have dropped Windows support. So for now, it's better to just keep it a more simple and "stupid" static response?

Reply:
Since which version we stopped support for Win?

Reply:
> Since which version we stopped support for Win?
I don't know actually. I'm hearing about it many times but I didn't find any official reference.

Reply:
@lukidzi @slonka 
Just created this new replacement PR, that one does not rely on the listener in the `kuma-dp` process:
https://github.com/kumahq/kuma/pull/11145

Reply:
I got two options:
* use named pipe
* listen a port and bind to `127.0.0.1`
Which one do you prefer?

Reply:
I prefer `listen a port and bind to 127.0.0.1` - if we ever need to get back to windows this will work there as well. @lukidzi - WDYT?

Reply:
Yeah, a port that binds to `127.0.0.1` makes sense to me. However, I don't like to make a new annotation to let the users specify this port (to prevent potential conflicts), which makes things complicated.

Reply:
I'll use `9902` for now, and please feel free to raise your ideas.

Reply:
We could bind to localhost and define the port. But it has to be configurable. If you run 2 instances of DP on the same VM you are going to hit conflict

Reply:
Thanks for the reply, I'm providing a configuration key to change the port.